README.md

```diff
@@ -12,11 +12,11 @@ Splink is a Python package for probabilistic record linkage (entity resolution)
 
 ## Key Features
 
-⚡ **Speed:** Capable of linking a million records on a laptop in approximately one minute.  
-🎯 **Accuracy:** Full support for term frequency adjustments and user-defined fuzzy matching logic.  
-🌐 **Scalability:** Execute linkage jobs in Python (using DuckDB) or big-data backends like AWS Athena or Spark for 100+ million records.  
-🎓 **Unsupervised Learning:** No training data is required, as models can be trained using an unsupervised approach.  
-📊 **Interactive Outputs:** Provides a wide range of interactive outputs to help users understand their model and diagnose linkage problems.  
+⚡ **Speed:** Capable of linking a million records on a laptop in approximately one minute.
```

Comment:
Gotta leave those spaces, they're needed for the markdown list formatting

Reply:
I think this is because of the linter. It has been fixed in master to not [lint the readme](https://github.com/moj-analytical-services/splink/blob/886089b749331117ed1c9db07fd0f2c87abaddde/scripts/lint_and_format.sh#L17), but I think this PR was opened before then. Hopefully this is the last PR where we to revert this

Reply:
Yeah, it's the linter annoyingly.

I'll revert the linting change...